src/js/api/users-api.js

```diff
@@ -34,6 +36,14 @@ const Api = {
         .set('Content-Type', 'application/json')
         .send(body)
         .end((err, res) => {
+          if (err && !res) {
+            var errorResponse2 = {
+              text: err.response ? JSON.parse(err.response.text) : err,
+              code: err.status
+            };
+            AppActions.setSnackbar(preformatWithRequestID(err.res, err.message), null, 'Copy to clipboard');
```

Comment:
The api/ non-view layer of the application should be kept free from GUI side effects to keep them a bit more stable than the ever changing components/ view layer...